<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>216</width>
    <height>158</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>216</width>
    <height>158</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>216</width>
    <height>158</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Captura de imagem</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../assets/icons/Dead_Human3.gif</normaloff>../assets/icons/Dead_Human3.gif</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="bt_adicionar">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Adicionar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_cancelar">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancelar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_capturar">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>110</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Capturar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_set_combo">
    <property name="geometry">
     <rect>
      <x>181</x>
      <y>50</y>
      <width>31</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../assets/icons/arrow.png</normaloff>../assets/icons/arrow.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>10</width>
      <height>10</height>
     </size>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>17</y>
      <width>77</width>
      <height>68</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: red</string>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>77</width>
       <height>68</height>
      </rect>
     </property>
     <widget class="QLabel" name="lb_screenshot">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>3</y>
        <width>61</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>81</width>
      <height>24</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777211</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Umpush</family>
     </font>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::PreventContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox {
    combobox-popup: 0;
}</string>
    </property>
    <property name="currentText">
     <string/>
    </property>
    <property name="maxVisibleItems">
     <number>24</number>
    </property>
    <property name="maxCount">
     <number>24</number>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
    </property>
    <property name="minimumContentsLength">
     <number>16</number>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../assets/resources.qrc"/>
 </resources>
 <connections/>
</ui>
